src/modules/Instabug.ts

```diff
@@ -547,18 +550,66 @@ export const addExperiments = (experiments: string[]) => {
 /**
  * Remove experiments from next report.
  * @param experiments An array of experiments to remove from the next report.
+ *
+ * @deprecated Please migrate to the new FeatureRequests APIs: {@link removeFeatureFlags}.
  */
 export const removeExperiments = (experiments: string[]) => {
   NativeInstabug.removeExperiments(experiments);
 };
 
 /**
  * Clear all experiments
+ *
+ * @deprecated Please migrate to the new FeatureRequests APIs: {@link removeAllFeatureFlags}.
  */
 export const clearAllExperiments = () => {
   NativeInstabug.clearAllExperiments();
 };
 
+/**
+ * Add featureFlags to next report.
+ * @param featureFlags An array of featureFlags to add to the next report.
+ */
+export const addFeatureFlags = (featureFlags: IBGFeatureRequest[]) => {
+  const flags: Record<string, string | undefined> = {};
+
+  for (const item of featureFlags) {
+    flags[item.name] = item.variant ? item.variant : '';
+  }
```

Comment:
Same as the `ReadableArray` comment on Android, why are we converting the array to an object and then to an array in the native side?

Reply:
since I send a map from the js side as I sent the feature flag as {name:" //name", variant:"// variant"}